tools/devops/automation/templates/release/vs-insertion-prep.yml

```diff
@@ -150,7 +150,7 @@ stages:
 
       - task: DownloadPipelineArtifact@2
         inputs:
-          artifactName: DropMetadata-$(Build.BuildId)-nugets-$(System.JobAttempt)
+          artifactName: DropMetadata-shipping-nugets
```

Comment:
What happens when there is a second run? AFAIK in the second run it will fail. I remember that this was added by @rolfbjarne  for a reason. Perse it looks good, I just want to make sure we are not missing a corner case.

Reply:
Since the upload is done in the new template context outputs section, it is the last step that will run as part of the job. I don't believe it will be possible for the job to fail before this artifact is created and have the artifact still be uploaded. There is also a new `dropMetadataPrefixName` parameter on the template that can be set if individual repos run into other corner cases or issues here and need more control over naming.